src/linear_algebra/clifford_algebra/equivs.lean

```diff
@@ -32,6 +61,35 @@ variables {R : Type*} [comm_ring R]
 lemma ι_eq_zero : ι (0 : quadratic_form R unit) = 0 :=
 subsingleton.elim _ _
 
+/-- Since the vector space is empty the ring is commutative. -/
+instance : comm_ring (clifford_algebra (0 : quadratic_form R unit)) :=
+{ mul_comm := λ x y, begin
+    induction x using clifford_algebra.induction,
+    case h_grade0 : r { apply algebra.commutes, },
+    case h_grade1 : x { simp, },
+    case h_add : x₁ x₂ hx₁ hx₂ { rw [mul_add, add_mul, hx₁, hx₂], },
+    case h_mul : x₁ x₂ hx₁ hx₂ { rw [mul_assoc, hx₂, ←mul_assoc, hx₁, ←mul_assoc], },
+  end,
+  ..clifford_algebra.ring _ }
+
+lemma reverse_apply (x : clifford_algebra (0 : quadratic_form R unit)) :
```

Comment:
```suggestion
@[simp] lemma reverse_apply (x : clifford_algebra (0 : quadratic_form R unit)) :
```

Reply:
The linter rejects that, as `reverse_eq_id` below is a stronger statement.